src/Components/Aspire.Seq/AspireSeqExtensions.cs

```diff
@@ -23,47 +24,41 @@ public static class AspireSeqExtensions
     /// <param name="builder">The <see cref="IHostApplicationBuilder" /> to read config from and add services to.</param>
     /// <param name="connectionName">A name used to retrieve the connection string from the ConnectionStrings configuration section.</param>
     /// <param name="configureSettings">An optional delegate that can be used for customizing options. It's invoked after the settings are read from the configuration.</param>
-    public static void AddSeqEndpoint(this IHostApplicationBuilder builder, string connectionName, Action<SeqSettings>? configureSettings = null)
+    public static void AddSeqEndpoint(
+        this IHostApplicationBuilder builder,
+        string connectionName,
+        Action<SeqSettings>? configureSettings = null)
     {
         ArgumentNullException.ThrowIfNull(builder);
 
         var settings = new SeqSettings();
+        settings.ServerUrl = builder.Configuration.GetConnectionString(connectionName);
+        settings.Logs.Protocol = OtlpExportProtocol.HttpProtobuf;
+        settings.Traces.Protocol = OtlpExportProtocol.HttpProtobuf;
+
         builder.Configuration.GetSection("Aspire:Seq").Bind(settings);
```

Comment:
The settings should continue to be bound to the configuration section before getting the connection string from `connectionName`. That is consistent with every other component. The order of precedence goes (lower number wins):

1. User code (`configureSettings`) always wins
2. ConnectionStrings w/ connection name
3. Configuration section

Reply:
Thanks. Done.